docs/maven-plugin.rst

```diff
@@ -6,8 +6,8 @@ A Maven plugin is available to help throughout the development process.
 schema-registry:download
 ========================
 
-The `download` goal is used to pull down schemas from a schema registry server. This goal can be used to
-This plugin is used to download AVRO schemas for the requested subjects and write them to a folder on the local file system.
+The `download` goal is used to pull down schemas from a schema registry server. This plugin is used to download AVRO
```

Comment:
`goal` instead of `plugin` is correct here because multiple `goal`s can be included in one maven `plugin`. The plugin is `schema-registry`, the `goal` is `download`.

Reply:
i'm a little confused here...so `This goal is used to download AVRO`?